android/modules/ui/src/java/ti/modules/titanium/ui/WindowProxy.java

```diff
@@ -639,6 +639,12 @@ private Transition createTransition(KrollDict props, String key)
 		return transition;
 	}
 
+	@Override
+	public String toString()
+	{
+		return "[object Window]";
+	}
+
```

Comment:
Do we need this? I just tested calling `toString()` on a `Ti.UI.Window` instance on 9.1.0 and it returned `"[object Window]"`.

On iOS, it returns `"[object TiUIWindow]"` which is the name of the objective-c class. That's an interesting inconsistency.

Reply:
For some reason this test failed with this PR: [ti.ui.window.test.js#L377](https://github.com/appcelerator/titanium_mobile/blob/master/tests/Resources/ti.ui.window.test.js#L377)
I realized we didn't override `toString()` on our KrollProxy, so I added that